src/Compiler/Microsoft.AspNetCore.Razor.Language/src/BoundAttributeDescriptor.cs

```diff
@@ -1,11 +1,12 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
File encoding changes :(. I thought I had fixed these...

Reply:
I think `encoding=UTF-8` is still missing from `.gitattributes` (https://github.com/dotnet/razor/pull/8099#issuecomment-1376904265).